content/index.pl.md

```diff
@@ -0,0 +1,223 @@
+      <td>Przykładowe narzędzie autorskie służące do tworzenia treści internetowych</td>
+    </tr>
+    <tr>
+      <td>Również przeglądarki internetowe, odtwarzacze mediów i aplikacje muszą obsługiwać funkcje dostępności.</td>
+      <td>Przykładowa przeglądarka internetowa używana do wyświetlania treści internetowych</td>
+    </tr>
+    <tr>
+      <td>Aby pomóc uczynić Internet dostępnym, W3C zapewnia standardy, które są uznawane na całym świecie przez rządy i firmy.</td>
+      <td>[Shadi mówi]</td>
+    </tr>
+    <tr>
+      <td>Najbardziej znanym z nich są Wytyczne dla dostępności treści internetowych - WCAG. WCAG to także norma ISO 40500, przyjęta również jako norma europejska EN 301 549. Opiera się na czterech podstawowych zasadach:</td>
+      <td>Wytyczne dotyczące dostępności treści internetowych — WCAG; ISO 40500; EN 301 549</td>
+    </tr>
+    <tr>
+      <td>Po pierwsze, postrzegalne - na przykład, aby ludzie mogli treść zobaczyć lub ją usłyszeć.</td>
+      <td>Ktoś pisze na tablecie i słucha tego przez słuchawki</td>
+    </tr>
+    <tr>
+      <td>Funkcjonalne - na przykład, aby ludzie mogli korzystać z komputera, pisząc na klawiaturze lub używając głosu.</td>
+      <td>Ktoś rozmawia z komputerem</td>
+    </tr>
+    <tr>
+      <td>Zrozumiałe - na przykład, aby treść były pisana jasnym i prostym językiem.</td>
+      <td>Dwie osoby zdezorientowane, patrzą na ekran komputera z gęstą stroną internetową</td>
+    </tr>
+    <tr>
+      <td>I solidne - aby ludzie mogli korzystać z różnych technologii wspomagających.</td>
+      <td>Ktoś używający powiększenia ekranu na dużym ekranie komputera</td>
+    </tr>
+    <tr>
+      <td>Oprócz WCAG, W3C udostępnia również Wytyczne dla dostępności narzędzi autorskich - ATAG, które określają wymagania dla systemów zarządzania treścią, edytorów kodu i innego oprogramowania.</td>
+      <td>Wytyczne dla dostępności narzędzi autorskich — ATAG</td>
+    </tr>
+    <tr>
+      <td>Z kolei Wytyczne dla dostępności programu użytkownika - UAAG, określają wymagania dla przeglądarek internetowych i odtwarzaczy multimedialnych.</td>
+      <td>Wytyczne dla dostępności programu użytkownika — UAAG</td>
+    </tr>
+    <tr>
+      <td>Na świecie żyje ponad miliard osób z niepełnosprawnością, czyli około 15-20% populacji. Konwencja ONZ o prawach osób z niepełnosprawnościami definiuje dostęp do informacji, w tym do Internetu, jako prawo człowieka. Większość krajów na całym świecie ratyfikowała tę konwencję ONZ, a kilka z nich przyjęło również wiążące polityki. Jednak niezależnie od wszelkich przepisów i regulacji, wdrażanie standardów dostępności jest niezbędne dla osób z niepełnosprawnościami i przydatne dla wszystkich.</td>
+      <td>[Shadi mówi]</td>
+    </tr>
+    <tr>
+      <td>Aby uzyskać więcej informacji na temat dostępności internetowej, odwiedź stronę w3.o-r-g/W-A-I</td>
```

Comment:
`w3.o-r-g/W-A-I` - myślniki są tutaj celowo? W oryginalnym tłumaczeniu ich nie ma.

Reply:
Tak. To jest translacja dla osób, które nie widzą. Myślniki są tu celowo, aby adres był zrozumiale ogłaszany przez czytnik ekranu.  Dla osób, które widzą, źródłem jest film.

Reply:
W innych tłumaczeniach tego nie spotkałem, ale tak właśnie myślałem, że taki jest cel.